### ISSUE TITLE
Could there be better support for textareas?

### ISSUE THREAD

**Issue by jamesrez** [kikoano/web2vr#12]
Right now I'm able to plug the textarea into VR but it's a bit limited.

My main issue is being able to select where in the textarea I want to edit. Seems like it defaults to the end of the text.

Is it possible to determine what part of the textarea I'm clicking on? 

I also notice the textarea isn't showing tab indented spaces?

Thanks for the help, this has been a great tool so far

**Comment by kikoano** [comment]
There is no feature to select input text cursor position. Will be nice if I can find some aframe component for it. I could try to make it one day.

**Comment by jamesrez** [comment]
Yeah I haven't found anything suitable yet. Not sure how aframe would handle figuring out where in the text you're at.
I'm going to keep looking / working at it. Thanks again

**Comment by kylebakerio** [comment]
Maybe left/right/up/down arrow keys in-world alongside an in-world keyboard would be an ok compromise?
